exec/vm.go

```diff
@@ -401,6 +459,17 @@ outer:
 	return 0
 }
 
+// Close frees any resources managed by the VM.
+func (vm *VM) Close() error {
+	if vm.nativeBackend != nil {
+		if err := vm.nativeBackend.Close(); err != nil {
+			return err
+		}
+	}
+	vm.abort = true // prevents further use.
```

Comment:
is it ok to leave this to `false` if `vm.nativeBackend.Close()` errors out?

Reply:
I guess not - some executable pages would be unmapped. I'll move it up.